homework-solutions/js/hw-5/task-3.js

```diff
@@ -1,10 +1,21 @@
+'use strict';
 /**
  * Создать строку с информацией о количестве гласных и согласных букв в слове.
  * Переменная `word` уже создана и содержит строку со словом.
  * Ожидаемый результат для `hello`: "hello contains 2 vowels and 3 consonants".
  */
 const word = 'hello';
+let vowels = 'aeiouy';
+let VowelsCount = 0;
+let ConsonantsCount = 0;
```

Comment:
```suggestion
let consonantsCount = 0;
```

С большой буквы называют классы, не переменные